### PR TITLE
Enhancement Repository Configuration and Optimize Render Script

### PR DESCRIPTION
**Pull request by michaelamattes**
- optimize render.sh and fix error from sed 
- use deeper hash values in build.yaml
- #55

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -42,52 +42,49 @@ touch build.yaml
 
 The DMC uses the wakemeops image as base image.
 
-| ARG          | Required | Default | Description                                                                                |
-| ------------ | -------- | ------- |------------------------------------------------------------------------------------------- |
-| DISTRIBUTION | no       | ubuntu  | distribution that should be used, see also [wakemeops](https://hub.docker.com/u/wakemeops) |
-| VERSION      | no       | latest  | specific version                                                                           |
+| ARG          | Required | Default | Description                                                                                | build.yaml use default | build.yaml setup custom |
```
Comment (rndmh3ro) [inline README.MD:45]: What do "use default" and "setup custom" mean?

File: README.MD [hunk 2]
```diff
@@ -42,52 +42,49 @@ touch build.yaml
...
+| REPOSITORIES_MICROSOFT_GPG   | no       | `https://packages.microsoft.com/keys/microsoft.asc`                                | defined default                                                                                                                  |                                                                                                                                                                        |                                                                                                                                                                                                                                |
```
Comment (rndmh3ro) [inline README.MD:64]: I don't understand what `defined default` means.

File: README.MD [hunk 3]
```diff
@@ -42,52 +42,49 @@ touch build.yaml
...
+| ARG                 | Required | Default | Description                                               | build.yaml use default | build.yaml setup custom                                                                                                                                |
+| ------------------- | -------- | ------- | --------------------------------------------------------- | ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| WORKDIR             | no       | /root   |                                                           |                        |                                                                                                                                                        |
+| PROFILES            | no       |         | configuration for profile settings                        |                        | profiles:<br>&nbsp;&nbsp;.vimrc:<br>&nbsp;&nbsp;&nbsp;&nbsp;- filetype on<br>&nbsp;&nbsp;.bash_aliases:<br>&nbsp;&nbsp;&nbsp;&nbsp;- alias ll='ls -la' |
```
Comment (rndmh3ro) [inline README.MD:86]: Can you put the "profiles"-code in codeblocks?

**Review by rndmh3ro — CHANGES_REQUESTED** [review]

File: README.MD (hunk 3 above)
Comment (michaelamattes) [reply README.MD:86]: I can't find a way to this within a table.

File: README.MD (hunk 2 above)
Comment (michaelamattes) [reply README.MD:64]: `predefined default value` is this better for the understanding ?

Comment (rndmh3ro) [reply README.MD:64]: just `default`.

**Review by rndmh3ro — APPROVED** [review]

**Pull request merged**